sechub-developertools/scripts/sechub-api.sh

```diff
@@ -168,6 +175,34 @@ function check_file {
 }
 
 
+function check_number {
+  local param="$1"
+  local parameter_name="$2"
+
+  check_parameter "$param" "$parameter_name"
+  if ! $failed ; then
+    if [[ ! ${!param} =~ ^[0-9]+$ ]]; then
+      echo "$parameter_name is not numeric."
+      failed=true
+    fi
+  fi
+}
+
+
+function check_time_unit {
+  param="$1"
```

Comment:
Ist es Absicht, dass bei function check_number "param" und "parameter_name" lokale Variablen sind, hier aber globale?

Reply:
`param` and `parameter` are only used inside the functions.
They all should be local. I'll update the rest of the functions.